### ISSUE TITLE
After wake up from sleep terminals go blank

### ISSUE THREAD

**Issue by ingemar-hn** [Eugeny/tabby#7969]
Version: 1.0.189 (and 1.0.188)
Platform: Linux x64 5.15.0-60-generic
Plugins: none
Frontend: xterm

After wake up from sleep all text with standard foreground color goes blank. The terminal is actually working, it's only the standard FG colored text that doesn't show. Running a command, e.g. ls, shows output where the text is in non-standard FG.
Zooming in or out restores the text, but going back to normal zoom hides the text again. This is new behavior from 1.0.188, in earlier versions the terminals were restored.
I'm using a light theme (Violet Light). I can't find any way to restore the terminal in normal zoom. Using the default theme Material works slightly better. Sometimes it works as it should, sometimes this workaround is needed: 1. Change zoom, 2. run a command, 3. Reset zoom. Now all text is shown.

Using WebGL with Material works fine. But it doesn't work well with Violet Light. In the latter case the text is very blurry and almost unreadable.

The problem has been around for a while, since issue #2843 at least.

**Comment by ingemar-hn** [comment]
I might have found the actual problem!
The FG color in Violet Light is #536870. I don't think that color matches any of the possible pre-selection alternatives in the Settings tab (i.e. any of the selectable color dots).
I've now saved a customized version of the theme with FG color set to #455A64 and it works (the test period is quite short however :-), the text in the terminals is restored after wake up from sleep. 
Using WebGL also works, the text isn't blurry anymore.

**Comment by ingemar-hn** [comment]
Sorry to say, but my above positive experiences were only temporary. I'm now back to previous behavior and I'll downgrade my installation to 187 again.

**Comment by konstantin83** [comment]
@ingemar-hn Have you found the solution yet? I have the same problem and can't find the colution for quite a lot of time.

**Comment by ingemar-hn** [comment]
No, no solution yet. But a workaround which isn't very good, but it usually works :)
I have two themes that I like, and couple of variants of these where the differences aren't really noticeable. Switching between these themes usually works (note that you in most cases have to press Enter to see if the switch worked).

**Comment by konstantin83** [comment]
It's a shame to keep seing such a strange problem after all this time...

**Comment by ingemar-hn** [comment]
Just out of curiosity, what brand of graphics card do you have?

**Comment by konstantin83** [comment]
NVIDIA Corporation GP104 [GeForce GTX 1070 Ti]

And CPU is AMD® Ryzen 9 3900xt 12-core processor × 24

**Comment by ingemar-hn** [comment]
Mine is NVIDIA GeForce GTX 1060. 
The reason I'm asking is that I a while ago noticed thread about a similar problem in VS Code. That problem was related to Nvidia cards.

**Comment by AdamJSoftware** [comment]
Also experiencing the issue. Manjaro with 1650 mobile gpu

**Comment by kshensovato** [comment]
I have the same issue. Ubuntu 22.04, Nvidia RTX A1000 laptop GPU. Oceanic Material theme.
